oe_theme.theme

```diff
@@ -65,8 +65,10 @@ function oe_theme_preprocess_menu__main(array &$variables): void {
   // Massage data to be compliant with ECL navigation menu data structure.
   $variables['links'] = array_map(function ($item) {
     return [
-      'href' => $item['url'],
-      'label' => $item['title'],
+      'link' => [
```

Comment:
This change seems out of scope. In fact the menu--main template uses a component that is no longer there in ecl2. So probably we should change this together with the cleanup of the menu template.

Reply:
Fair enough, I remove.